recipes/ta-numba/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: ta-numba
+  version: "0.2.2"
+
+source:
+  url: https://pypi.io/packages/source/t/ta-numba/ta_numba-0.2.2.tar.gz
```

Comment:
Please use a jinja variable for the version. See:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L9-L16

In general, please look at our example recipe for how we like the recipe written and possibly use:
https://github.com/conda/grayskull